python/lsst/meas/extensions/trailedSources/NaivePlugin.py

```diff
@@ -134,6 +134,7 @@ def __init__(self, config, name, schema, metadata, logName=None):
         self.NAN = flagDefs.add("flag_nan", "One or more trail coordinates are missing")
         self.SUSPECT_LONG_TRAIL = flagDefs.add("flag_suspect_long_trail",
                                                "Trail length is greater than three times the psf radius")
+        self.SHAPE = flagDefs.add("flag_shape", "Shape flag is set, trail not calculated")
```

Comment:
Do you mean trail **length** is not calculated?

Reply:
Yes I do. Changed.